lib/nfg_ui/bootstrap/utilities/modalable.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+module NfgUi
+  module Bootstrap
+    module Utilities
+      attr_reader :remote
```

Comment:
@hoenth this attr_reader isn't necessary (As far as I can tell from some testing), but... it does add clarity as to the fact that `remote` will be a part of this module working as intended. Seeking advice on what best practice is here.